ZeldaFullEditor/Gui/DungeonMain.cs

```diff
@@ -2635,7 +2711,7 @@ private void TabControl2_SelectedIndexChanged(object sender, EventArgs e)
                 }
                 else
                 {
-                    // TODO is this wrong?
+                    // TODO: Is this wrong?
```

Comment:
The effort in adding the colon and capital punctuation is cute but nit: vague todo

Reply:
🤣 and again i'm just doin my job, i have no idea what this comment is for lol